src/defaultsettings.cpp

```diff
@@ -321,27 +323,27 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("movement_liquid_sink", "10");
 	settings->setDefault("movement_gravity", "9.81");
 
-	//liquid stuff
+	// Liquids
 	settings->setDefault("liquid_loop_max", "100000");
 	settings->setDefault("liquid_queue_purge_time", "0");
 	settings->setDefault("liquid_update", "1.0");
 
-	//mapgen stuff
+	// Mapgen
 	settings->setDefault("mg_name", "v7");
 	settings->setDefault("water_level", "1");
 	settings->setDefault("chunksize", "5");
 	settings->setDefault("mg_flags", "dungeons");
+	settings->setDefault("mgv6_spflags", "jungles, snowbiomes");
```

Comment:
Where did this come from? It's not present in current settings and not present in MTGame .conf.